extensions/intellij/intellij_generator_plugin/src/main/resources/templates/rx_bloc_tests/bloc_alchemist_golden.dart.template

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter_test/flutter_test.dart';
+
+import '../../helpers/golden_helper.dart';
+import '../../helpers/models/scenario.dart';
+import '../factory/${bloc_snake_case}_factory.dart';
+import 'package:rx_bloc/rx_bloc.dart';
+import 'package:rx_bloc_list/models.dart';
```

Comment:
Do we really need to import the rx_bloc_list and rx_bloc ?

Reply:
All unused imports were removed.